packages/react-search-ui/src/__tests__/SearchConsumer.test.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { mount } from "enzyme";
+
+import { SearchProvider, SearchConsumer } from "../..";
+
+describe("SearchConsumer", () => {
+  it("exposes state and actions to components", () => {
```

Comment:
Should we test that we expose actions or is testing that we expose state is enough?

Reply:
Good point, I think we should test this.

Reply:
I have a `will inject a specified action` test for this.

Reply:
Done.